### PR TITLE
7za: fix Japanese translation

### PR DESCRIPTION
**Pull request by Ry0taK**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Japanese translation of 7za were bit broken (e.g. `道/に/ファイルまたはディレクトリ` means `File or directory on the road`), so I tweaked it a bit.

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]
Thank you for contribution.

File: pages.ja/common/7za.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 # 7za
 
-> 7-Zipは高圧縮率のファイルアーカイバ
-> より少ないアーカイブタイプをサポートする `7z`. スタンドアロンバージョン.
-> 詳しくはこちら: <https://www.7-zip.org>.
+> 圧縮率の高いファイルアーカイバ
+> `7z`よりも対応しているファイル形式は少ないですが、複数のプラットフォームに対応しています。
```
Comment (blueskyson) [inline pages.ja/common/7za.md:4]: Please put spaces between full-width characters and Latin characters.
```suggestion
> `7z` よりも対応しているファイル形式は少ないですが、複数のプラットフォームに対応しています。
```

Comment (Ry0taK) [reply pages.ja/common/7za.md]: Thank you for the review! I've applied the suggestion.

File: pages.ja/common/7za.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
 # 7za
 
-> 7-Zipは高圧縮率のファイルアーカイバ
-> より少ないアーカイブタイプをサポートする `7z`. スタンドアロンバージョン.
-> 詳しくはこちら: <https://www.7-zip.org>.
+> 圧縮率の高いファイルアーカイバ
```
Comment (navarroaxel) [inline pages.ja/common/7za.md]:
```suggestion
> 圧縮率の高いファイルアーカイバ。
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**